src/components/BorrowerProfile/BorrowerCountry.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+	<article>
+		<kv-map
+			class="
+				tw-rounded
+				tw-overflow-hidden
+				tw-my-5
```

Comment:
Great job getting this component in place @mlibazisi. The only thing I updated in this file was to change the margin to only vertical and bump it up once more.